app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -1144,7 +1177,11 @@ class BrowserTabFragment :
         super.onResume()
 
         if (viewModel.hasOmnibarPositionChanged(omnibar.omnibarPosition)) {
-            requireActivity().recreate()
+            if (swipingTabsFeature.isEnabled) {
+                (requireActivity() as BrowserActivity).clearTabsAndRecreate()
```

Comment:
This code can crash the Custom tab. See scenario:
- open the DDG app with the Address bar set at the top
- open a DDG custom tab from another app
- go back to the DDG app and change the Address bar to the bottom
- go back to the DG custom tab and see it crashes as there is no BrowserActivity there

Reply:
Nice catch, thanks! It’s fixed [here](https://github.com/duckduckgo/Android/pull/5408/commits/803d97353f61b3415f9d6c851b87699211122a03).